loadable_apps/loadable_sample/wifiapp/wifiapp.c

```diff
@@ -49,6 +49,10 @@ static void display_test_scenario(void)
 
 extern int preapp_start(int argc, char **argv);
 
+#ifdef CONFIG_EXAMPLES_SMARTFS_POWERCUT
+extern int smartfs_powercut_main(int argc, char *argv[]);
```

Comment:
Calling powercut testcase from Wifi app?
Could you please explain why exactly?

Reply:
If we use loadable application, main binary is wifi (cpu) another one is micom.
so I loadable set, this will be excuted otherwise, will use entrypoint of application config.